src/Jackett.Common/Definitions/freetorrent.yml

```
@@ -16,7 +16,7 @@ caps:
     - {id: 1, cat: Movies, desc: "Movies"}
     - {id: 2, cat: TV, desc: "TV"}
     - {id: 3, cat: Audio, desc: "Music"}
-    - {id: 4, cat: PC/0day, desc: "Software"}
+    - {id: 4, cat: PC/Software, desc: "Software"}
```

the **Category Names** must match the torznab specifications
See this list for valid options:
https://github.com/nZEDb/nZEDb/blob/dev/docs/newznab_api_specification.txt#L608-L695
Also, while you are at it, the web site has 3 new categories you can add to the list
```suggestion
    - {id: 4, cat: PC/0day, desc: "Software"}
    - {id: 12, cat: TV/Documentary, desc: "Documentaries"}
    - {id: 13, cat: TV, desc: "Shows"}
    - {id: 14, cat: Audio/Video, desc: "Clips musicaux"}
```